src/Draco.RedGreenTree.SourceGeneration/SymbolExtensions.cs

```diff
@@ -0,0 +1,12 @@
+using Microsoft.CodeAnalysis;
+
+namespace Draco.RedGreenTree.SourceGeneration;
+
+internal static class SymbolExtensions
+{
+    private static readonly SymbolDisplayFormat displayFormat =
+        new(typeQualificationStyle: SymbolDisplayTypeQualificationStyle.NameAndContainingTypesAndNamespaces);
+
+    public static string GetNamespacedName(this INamedTypeSymbol type) =>
+        type.ToDisplayString(displayFormat);
```

Comment:
Isn't this the default behavior already?

Reply:
Nope, although I think it's the default behavior of `SymbolDisplayFormat.CSharpErrorMessageFormat`. Just thought it'd be best to be explicit (and also provide a much shorter name, `x.ToDisplayString(SymbolDisplayFormat.CSharpErrorMessageFormat)` -> `x.ToNamespacedName()`).